Source/Private/CommandCoreResult.ps1

```diff
@@ -0,0 +1,8 @@
+class CommandCoreResult {
```

Comment:
Is there any reason for the rename? Seems unnecessary if you have removed the old `CommandResult`
```suggestion
class CommandResult {
```

Reply:
I don't think there is any reason other than @zionyx did it before I took over the work on this branch.

Reply:
But your point is valid. I have changed it so the name is now `CommandResult`